src/pages/dev/send-tokens/interchain-tokens/interchain-token-executable.mdx

```diff
@@ -0,0 +1,115 @@
+# Interchain Token Service Executable
+
+import { Callout } from "/src/components/callout";
+
+You can use the [Axelar Interchain Token Service (ITS)](https://github.com/axelarnetwork/interchain-token-service/tree/main) to create a new Interchain Token or upgrade existing ERC-20 tokens to an [Interchain Token](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interchain-token/InterchainToken.sol). ITS also allows you to send messages along with Interchain Tokens across different blockchain networks.
+
+The Interchain Token Service is deployed to [`0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C`](https://etherscan.io/address/0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C) while the Interchain Token Factory is deployed to [`0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66`](https://etherscan.io/address/0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66).
+
+## Deploy a new Interchain Token
```

Comment:
Isn't this duplicative with the other pages? Shouldn't this page just focus on sending a message and building an executable contract with an existing token and refer to the other page for this?